azurerm/resource_arm_express_route_circuit.go

```diff
@@ -62,7 +62,6 @@ func resourceArmExpressRouteCircuit() *schema.Resource {
 							Required: true,
 							ValidateFunc: validation.StringInSlice([]string{
 								string(network.Standard),
-								string(network.Premium),
```

Comment:
is this SKU still available? if so, should we add this back in as a hard-coded string?